content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst

```diff
@@ -1,54 +1,209 @@
 
-Now, if I purchase apples and carrots to my supplier, they will be grouped in the same receipt but
-redirected to the right location automatically, thanks to putaway rules. This information is
-available from *Inventory Report*, under the reporting menu.
+Receive the incoming shipment by selecting :menuselection:`Receive products --> Validate`.
 
-.. image:: putaway/putaw3.png
+.. image:: putaway/receive-incoming-stock.png
    :align: center
+   :alt: Receive package of apples and carrots
+
+Verify stock moves
+==================
 
-.. image:: putaway/putaw4.png
+Confirm whether the putaway rule worked as intended and that the products have been moved to the
+correct location in the :guilabel:`Inventory Report`. View all products in stock in
+:menuselection:`Inventory --> Reporting --> Inventory Report`. By default, the products are
+grouped by location.
+
+Select the product, and the drop-down displays the locations the products currently are. In the
+image below, the locations shown are `WH/Stock/Fruits` and `WH/Stock/Vegetables`, which
+confirms that the putaway rules successfully.
+
+.. image:: putaway/destination-locations.png
    :align: center
+   :alt: Verify stock moves triggered by putaway rule
+
+For a more detailed report, click on a stock move line. On the report, the :guilabel:`Reference`
+displays the warehouse operation associated to the :guilabel:`Product`. These can be any operation
+involving movement of stock, such as manufacturing orders, inventory adjustments, or internal
+transfers. The :guilabel:`Source` links the document associated to the reference operation. Use the
+:guilabel:`From` and :guilabel:`To` location and the associated documents to trace the path of the
+product.
 
-.. image:: putaway/putaw5.png
+.. _inventory/putaway/wrong-location:
+
+Trace products in wrong locations
+---------------------------------
+
+Pictured below is a detailed stock move of the :guilabel:`Product`, `Apple` that was put away in the
+wrong location. This report is useful to trace the path the product moved and identify the cause of
+the wrong stock move.
+
+Start by selecting the stock move record for `Apple`. Here, a putaway rule was purposely set up
+incorrectly, so the product was moved from `WH/Stock` to `WH/Stock/Shelf 1` instead of the expected
+location, `WH/Stock/Fruits`.
+
+.. image:: putaway/stock-move-report.png
    :align: center
+   :alt: Open stock move report
+
+The :guilabel:`To` field displays where the product currently is, `WH/Stock/Shelf 1`, which is the
+wrong location. The :guilabel:`From` location is the vendor's location, which is expected since the
+:guilabel:`Source` of the stock move is a :abbr:`PO (Purchase Order)`.
+
+Diagnose why the apple ended up in the shelf by following the :guilabel:`Reference` warehouse
+operation in :menuselection:`Inventory --> Operations --> Transfers`. This is the page that displays
+all warehouse transfers-- internal or external. Click on the desired record.
```

Comment:
We want to avoid directly referencing the images in the text, so this section should be rewritten to be more general and not refer specifically to the images.